docs/options.md

```diff
@@ -156,3 +156,12 @@ or
 Default | CLI Override | API Override
 --------|--------------|-------------
 `false` | `--require-pragma` | `requirePragma: <bool>`
+
+## Insert Pragma
```

Comment:
The docs should port to `README.md` as well, since we haven't implemented the docs synchronization (#2825).